app/assets/javascripts/application.js

```diff
@@ -58,10 +58,16 @@ function onContentLoad(){
    });
 
   // adds buttons classes to all links
-  $("#title_action a").addClass("btn btn-default");
+  $("#title_action a").each(function(index, item) {
+    $(item).addClass("btn");
+    // This conditional allows the documentation buttons to stand out
+    if (!$(item).hasClass('btn-info') && !$(item).hasClass('btn-danger')) {
+      $(item).addClass("btn-default");
+    }
+  });
```

Comment:
The last 5 lines can be written much more clearly and easily as
`$("#title_action a").addClass('btn').not('.btn-info, .btn-danger').addClass("btn-default");`